src/app/components/schedule-card/ScheduleCard.jsx

```diff
@@ -2,20 +2,18 @@ import "./ScheduleCard.scss";
 
 import { scheduleData } from "../../data/schedule-info";
 
-const Event = ({ name, description, time, tooltip }) => (
+const Event = ({ name, description, time, tooltip}) => (
 	<div className="schedule-card-time-slot">
 		<div>
-			<h5>{name}</h5>
+			<h5 className="name">{name}</h5>
+			<h5 style={{paddingTop: "2px"}}className="time">{time}</h5>
```

Comment:
we can prolly move this to the scss file now that we're done experimenting with it